react/SellingPrice.tsx

```diff
@@ -79,6 +80,9 @@ function SellingPrice({
 
   const taxValue = commercialOffer.Tax
 
+  const sellingPriceWithTaxWithoutUnit =
+    sellingPriceValue + (sellingPriceValue * taxPercentage / unitMultiplier)
```

Comment:
This is the correct `sellingPriceWithTax` calculation, so it seems like there's no need to create another property for it. I suggest altering the `sellingPriceWithTax` value to what would be `sellingPriceWithTaxWithoutUnit` as it would properly represent the selling price of the product with taxes.

Reply:
Hi @icazevedo 
The issue is that the commercial offer Tax that is used in the sellingPriceWithTax will return the tax for the entire multiplied value of the product. 

If there is a product that costs 100$ and has a 10% tax and the product has a 5.0 unit multiplier then the commercialOffer.Tax will return the tax for all 5 units.
10% of 100 = 10
commercialOffer.Tax = 50

So when displaying the price of a product without the unit multiplier but with the tax, that value will be wrong (150$ instead of 110$)

Please check the description of this PR, and also the video recording I attached.

You can test on [this workspace](https://mihaitest--vtexromania.myvtex.com/rama-foto-argintata-jardin-d-eden-christofle/p?__bindingAddress=vtexromania.myvtex.com/) to see the variable.
on the master workspace you will see the sellingPriceWithTax variable.

https://github.com/vtex-apps/product-price/assets/84322200/e39ff14b-0531-4402-b14a-841080808c0e

Reply:
Regarding modifying the sellingPriceWithTax it would mean modifying the commercialOffer.Tax and I think that would impact existing implementations. This way it will not be a breaking change. Maybe there are some use cases where the original sellingPriceWithTax calculation is used.